app/views/pages/notebooks.html.erb

```diff
@@ -0,0 +1,79 @@
+<% provide(:title, 'Getting Started with the Archives Unleashed Cloud Jupyter Notebooks') %>
+
+<% cache do %>
+  <div class="container">
+    <h1 class="about_h1">Getting Started with the Archives Unleashed Cloud Jupyter Notebooks</h1>
+    <p class="about_p_italic">Tutorial by Sarah McTavish (University of Waterloo)</p>
+    <div class="toc">
+        <ul class="toc_list">
+          <li class="toc-li"><%=link_to('Introduction', anchor: '1toc') %></li>
+          <li class="toc-li"><%=link_to('Getting Started', anchor: '2toc') %></li>
+          <li class="toc-li"><%=link_to('Using the Notebooks', anchor: '3toc') %></li>
+          <li class="toc-li"><%=link_to('Generating the Derivatives', anchor: '4toc') %></li>
+          <li class="toc-li"><%=link_to('User Configuration', anchor: '5toc') %></li>
+        </ul>
+    </div>
+    <br />
+    <h3 class="about_h3" id="1toc">Introduction</h3>
+    <p class="about_p">The <%= link_to('Archives Unleashed Cloud Notebooks', 'https://github.com/archivesunleashed/auk-notebooks', target: '_blank') %> require the use of the <%= link_to('Anaconda Distribution','https://www.anaconda.com/distribution/', target: '_blank') %>, as well as a working knowledge of the command line. For an introduction to the command line, see Ian Milligan and James Baker’s <%= link_to('Introduction to the Bash Command Line', 'https://programminghistorian.org/en/lessons/intro-to-bash', target: '_blank') %>.</p>
+    <p class="about_p">The Cloud Notebooks also require the use of the Archives Unleashed Cloud and an Archive-It account, in order to run and process Archives Unleashed WARC files. The Archives Unleashed Cloud <%= link_to('documentation', '/documentation', target: '_blank') %> will get you started on creating an account and linking your Archive-It account to the Archives Unleashed Cloud. We will be using the Archives Unleashed Cloud in order to generate the derivatives used with the Cloud Notebooks for text, domain, and network analysis.</p>
+    <p class="about_p">This tutorial was originally created to be used with the Mac OS. There are minor differences to running commands on Windows or Linux, as per the <%= link_to('lesson here', 'https://programminghistorian.org/en/lessons/intro-to-bash', target: '_blank') %>. Once the Cloud notebooks have been cloned to your system, the minor operating system differences should disappear!</p>
```

Comment:
suggested change: "This tutorial was originally created to be used with the Mac OS...." -->

"This tutorial was created using a Mac OS. There are minor differences when running commands on Windows or Linux, as per the lesson here."